operations/http.go

```diff
@@ -253,24 +252,38 @@ func (ac *legacyClient) GetPatchedConfig(patchId string) (*model.Project, error)
 	return ref, nil
 }
 
-// GetVersionConfig fetches the config requests project details from the API server for a given project ID.
-func (ac *legacyClient) GetConfig(versionId string) (*model.Project, error) {
+// GetVersionConfig fetches the config yaml from the API server for a given project ID.
+func (ac *legacyClient) GetConfig(versionId string) ([]byte, error) {
 	resp, err := ac.get(fmt.Sprintf("versions/%v/config", versionId), nil)
 	if err != nil {
 		return nil, err
 	}
 	if resp.StatusCode != http.StatusOK {
 		return nil, NewAPIError(resp)
 	}
-	ref := &model.Project{}
-	yamlBytes, err := ioutil.ReadAll(resp.Body)
+	respBytes, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "error reading body")
 	}
-	if _, err := model.LoadProjectInto(yamlBytes, "", ref); err != nil {
+	return respBytes, nil
+
```

Comment:
Note to self: this function only changes the return value BECAUSE the functions using it now don't need a project, they need the byte string to write to a function, so we aren't changing the route.